File: Solved/Sadegane Khaste/Sadegane Khaste.py
# صدگان خسته
# ID : 3406
# https://quera.ir/problemset/contest/3406/%D8%B3%D8%A4%D8%A7%D9%84-%D9%BE%DB%8C%D8%A7%D8%AF%D9%87-%D8%B3%D8%A7%D8%B2%DB%8C-%D8%B5%D8%AF%DA%AF%D8%A7%D9%86-%D8%AE%D8%B3%D8%AA%D9%87
# https://b2n.ir/957810

a = input()
b = input()

if a == b:
    print(a, "=", b)

elif int(a[::-1]) > int(b[::-1]):
    print(b, "<", a)

else:
    print(a, "<", b)

File: Solved/Sadegane Khaste/SadeganeKhaste.java
import java.util.*;

/*
Question title = Sadgane khaste - صدگان خسته
https://quera.ir/problemset/3406/
 */

public class SadeganeKhaste {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String strInput = sc.nextLine();
        String strInput2 = sc.nextLine();
        char[] input1 = strInput.toCharArray();
        char[] input2 = strInput2.toCharArray();
        List<Character> list = new ArrayList<>();
        List<Character> list2 = new ArrayList<>();

        for (char i : input1) {
            list.add(i);
        }
        for (char i : input2) {
            list2.add(i);
        }

        Collections.reverse(list);
        Collections.reverse(list2);
        int num1 = toString(list);
        int num2 = toString(list2);

        if (num1 > num2) {
            System.out.println(strInput2 + " < " + strInput);
        } else if (num1 < num2)
            System.out.println(strInput + " < " + strInput2);
        else {
            System.out.println(strInput + " = " + strInput2);
        }
    }

    private static int toString(List<Character> list) {
        StringBuilder sb = new StringBuilder();
        for (char i : list) {
            sb.append(i);
        }
        return Integer.parseInt(sb.toString());
    }
}